specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -583,6 +583,108 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/restorableDroppedSqlpools/{restorableDroppededSqlPoolId}": {
```

Comment:
How do you restore these? Also, typically these soft-delete scenarios give users the ability to purge the restorable objects by calling a DELETE API on the "restorable" variant.

Reply:
SO there are 2 options:
('les common)
1. user calls this API (visa some rest client, or via commandlet) and reaches out to CSS who creates an ICM  and we can help the user out

2. There is going to be an Azure portal blade which will call this API to show a drop down of the restorabled droppedpools. The user chooses one of them and the chosen one gets passed onto a Create from 'restorable dropped sql pool' flow., to create a new sql pool.

Reply:
What API is used in #2?  How do you specify via the API that you want to restore one of these databases?  Where is that API change?

Reply:
It would have to be the 'create sqlpool' API.  ( sqlpool.json file) with some update to it (to take some more parameters as per my understanding).

This line below if from a Colleague:
'o	Synapse RP would need to expose additional parameters (deletionTime, deletionDateString, deletionTimeString, deletedDatabaseId) in their create sqlpool api (to support the provided arm template)'